README.md

```diff
@@ -764,6 +764,10 @@ See [this guide](docs/assessment.md) for more details.
 Proceed to the [group migration workflow](#group-migration-workflow) below or go back to the
 [migration process diagram](#migration-process).
 
+The UCX assessment workflow is intended to only run once; re-running it is not supported. If the inventory and findings
```

Comment:
We should say something like, rerunning it will not clear the results of the last run.
Users may need to run assessment in certain use cases.
The instruction should be.
a. Avoid
b. If need to, clear the table that needs to be refreshed.

Reply:
I have updated the docs with a part of your comment. 

The advice (as discussed with @nfx) is to reinstall ucx to clear the tables, as stated in the change

Reply:
KK